stream/stream_test.go

```diff
@@ -111,79 +116,77 @@ var _ = Describe("Streaming", func() {
 				streamers[i] = NewStreamRecycler(NewStreamer(multiAddrs[i].Address(), &clients[i], &servers[i]))
 			}
 
-			for conns := 0; conns < 4; conns++ {
-				for i := 0; i < 128; i++ {
-					for j := 0; j < 128; j++ {
-						if i == j {
-							continue
-						}
-						_, err := streamers[i].Open(context.Background(), multiAddrs[j])
+			cancelConns := map[int]map[int]map[int]context.CancelFunc{}
+			for i := 0; i < 128; i++ {
```

Comment:
Have a constant or var set for the magic number 128.

Reply:
Fixed.